src/services/git/VCSServicesUtils.ts

```diff
@@ -5,29 +5,37 @@ import { GitHubPullRequestState } from './IGitHubService';
 export class VCSServicesUtils {
   static getPRState = (state: PullRequestState | undefined, service: VCSServiceType) => {
```

Comment:
Are we still using it somewhere? I'm just wondering if it is necessary to have it.

Reply:
removed